UPGRADE-2.x.md

```diff
@@ -9,3 +9,10 @@ UPGRADE FROM 2.0 to 2.1
 All files under the ``Tests`` directory are now correctly handled as internal test classes. 
 You can't extend them anymore, because they are only loaded when running internal tests. 
 More information can be found in the [composer docs](https://getcomposer.org/doc/04-schema.md#autoload-dev).
+
+UPGRADE FROM 2.1.1 to 2.1.2
```

Comment:
I think this has to be placed on top with a message like:

UPGRADE FROM 2.1.1 to 2.x

And when we do a release, we change it to the new version (it could be 2.1.2 or 2.2.0)